mitodl/hugo-course-publisher
# Left nav hierarchy UI

#### Pre-Flight checklist


#### What are the relevant tickets?
UI part of https://trello.com/c/JkknCyTL/11-hierarchical-left-nav

#### What's this PR do?
Adds a recursive menu_item.html partial that recursively renders all menu items passed in as a collapsible nav using bootstrap collapse classes

#### How should this be manually tested?
Run the site, browse courses that have child nodes (the analytics edge is a good example) and verify that child menu items are rendered and it looks like the design

#### Any background context you want to provide?
The child menu items having all the same name in some cases is a known issue: https://github.com/mitodl/ocw-to-hugo/issues/23

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/12089658/74958775-738e4880-53d7-11ea-933c-80b1a8208933.png)

![image](https://user-images.githubusercontent.com/12089658/74958697-578aa700-53d7-11ea-8ceb-f54de8a2e348.png)
